index.bs

```diff
@@ -715,6 +719,15 @@ A [=sensor=] has an associated <dfn>periodic reporting mode flag</dfn> which is
 
 A [=sensor=] has an associated <dfn>current sampling frequency</dfn> which is initially null.
 
+The [=current sampling frequency=] for a non[=set/is empty|empty=] [=ordered set|set=] of
```

Comment:
I'd rephrase as "For a non[=set/is empty|empty=] [=ordered set|set=] of [=activated sensor objects=] the [=current sampling frequency=] is equal to.."